src/definitions/constants.ts

```diff
@@ -39,6 +39,7 @@ export const VSCODE_QUARKUS_EXTENSION_NAME = "redhat.vscode-quarkus";
 
 export const DEFAULT_API_URL: string = 'https://code.quarkus.io/api';
 export const DEFAULT_BUILD_TOOL: BuildToolName = BuildToolName.Maven;
+export const DEFAULT_PLATFORM_VERSION: string = 'io.quarkus.platform:2.2';
```

Comment:
I don't see this constant being used anywhere. In fact when computing the the versions to populate the dropdown, you use the "recommended" value from the API request, so it looks like there's no need for this.

I see this as a good thing because this value would have had to be updated everytime a new version were released anyways.